packages/front-end/components/PowerCalculation/stats.ts

```diff
@@ -340,3 +356,294 @@ export function powerMetricWeeks(
   };
   return results;
 }
+
+/*******************/
+export function calculatePriorMean(
+  priorMeanRel: number,
+  mean: number,
+  relative: boolean
+): number {
+  return relative ? priorMeanRel : priorMeanRel * Math.abs(mean);
+}
+
+export function calculatePriorVariance(
+  priorVarianceRel: number,
+  mean: number,
+  relative: boolean
+): number {
+  return relative
+    ? priorVarianceRel
+    : priorVarianceRel * Math.pow(Math.abs(mean), 2);
```

Comment:
nit: is there a reason why we do Math.abs() first?

Reply:
removed